### ISSUE TITLE
please add the copyright attribution

### ISSUE THREAD

**Issue by BrunoBonacci** [bowbahdoe/flake#1]
Hi,

as part of the Apache 2.0 licensing, could you please add the following copyright attribution on every file you copied from mulog 

```
// ------------------------------------------------------------------------------------------
//  Copyright © 2019-2024 Bruno Bonacci 
//  source: https://github.com/BrunoBonacci/mulog
// ------------------------------------------------------------------------------------------
```

thank you

**Comment by bowbahdoe** [comment]
sure

**Comment by bowbahdoe** [comment]
Added attributions and published a new version `2024.08.21` so it will be in the sources jar. LMK if you want anything else.
